src/khoj/main.py

```diff
@@ -126,6 +127,24 @@ def run(should_start_server=True):
     # Setup task scheduler
     poll_task_scheduler()
 
+    # Setup Background Scheduler
+    from django.conf import settings as django_settings
+
+    django_db = django_settings.DATABASES["default"]
+    state.scheduler = BackgroundScheduler(
+        {
+            "apscheduler.jobstores.default": {
+                "type": "sqlalchemy",
```

Comment:
Should this be django?
```suggestion
                "type": "django",
```

Reply:
APScheduler automatically comes with a SQLAlchemy job store but not a Django job store. So to avoid adding another dependency like `django-apscheduler` to Khoj I decided to stick with using the default SQLAlchemy job store to connect with the Khoj DB